wagtail/admin/views/generic/mixins.py

```diff
@@ -203,13 +206,17 @@ def save_instance(self):
         return instance
 
     def get_success_message(self):
-        return _(
-            "%(model_name)s '%(instance)s' has been replaced with version from %(timestamp)s."
-        ) % {
-            "model_name": capfirst(self.model._meta.verbose_name),
-            "instance": self.object,
-            "timestamp": self.revision.created_at.strftime("%d %b %Y %H:%M"),
-        }
+        message = (
+            "{model_name} '{instance}' has been replaced with version from {timestamp}."
+        )
+        if self.action == "publish":
+            message = "Version from {timestamp} of {model_name} '{instance}' has been published."
+
+        return _(message).format(
```

Comment:
We should put the ``_()`` around the individual strings above so that ``makemessages`` can extract them

Reply:
Thanks! Fixed